src/Microsoft.ApiDesignGuidelines.Analyzers/Core/IdentifiersShouldNotMatchKeywords.cs

```diff
@@ -1,69 +1,621 @@
-                                                                             isEnabledByDefault: false,
+                                                                             Category,
+                                                                             Severity,
+                                                                             isEnabledByDefault: IsEnabledByDefault,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
-                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+                                                                             helpLinkUri: HelpLinkUri,
+                                                                             customTags: CustomTags);
         internal static DiagnosticDescriptor NamespaceRule = new DiagnosticDescriptor(RuleId,
                                                                              s_localizableTitle,
                                                                              s_localizableMessageNamespace,
-                                                                             DiagnosticCategory.Naming,
-                                                                             DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             Category,
+                                                                             Severity,
+                                                                             isEnabledByDefault: IsEnabledByDefault,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
-                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+                                                                             helpLinkUri: HelpLinkUri,
+                                                                             customTags: CustomTags);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(MemberParameterRule, MemberRule, TypeRule, NamespaceRule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
-            
+            analysisContext.RegisterSymbolAction(AnalyzeMemberRule,
+                SymbolKind.Event, SymbolKind.Method, SymbolKind.Property);
+
+            analysisContext.RegisterSymbolAction(AnalyzeMemberParameterRule,
+                SymbolKind.Method);
```

Comment:
Will this miss indexer parameters? Or do symbol actions run directly on the getter/setter?